internal/livelint/interaction.go

```diff
@@ -0,0 +1,60 @@
+package livelint
+
+import (
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+//go:generate moq -out mock_ui_test.go -pkg livelint_test . UserInteraction
+
+type UserInteraction interface {
+	DisplayVerboseSetting(verbose bool)
+	DisplayContext(contextMsg string)
```

Comment:
What all types of message do we want to differentiate? There seems to be a context message type, are there other types we want to show the user? I.e. do we need other types or some kind of message level to differentiate between different message types?

Reply:
I agree that we might need that in the future. However, I would add it when we need it.